src/components/WalletModal/Option.tsx

```diff
@@ -62,7 +62,7 @@ const CircleWrapper = styled.div`
 
 const HeaderText = styled.div`
   ${({ theme }) => theme.flexRowNoWrap};
-  color: ${props => (props.color === 'blue' ? ({ theme }) => theme.primary1 : ({ theme }) => theme.text1)};
+  /* color: ${props => (props.color === 'blue' ? ({ theme }) => theme.primary1 : ({ theme }) => theme.text1)}; */
```

Comment:
Disabling the color definition as it inherits the right light/dark mode for now.

Reply:
these should be modded as well